docs/EN_US/ConfiguringHPCC/ConfiguringHPCC.xml

```diff
@@ -1929,6 +1932,77 @@ sudo -u hpcc cp /etc/HPCCSystems/source/NewEnvironment.xml /etc/HPCCSystems/envi
                         xmlns:xi="http://www.w3.org/2001/XInclude" />
           </para>
         </sect3>
+
+        <sect3 id="CFGESP_WsStore" role="brk">
+          <title>Ws_Store</title>
+
+          <para>You can choose to add the ws_store ESP Service to your HPCC
```

Comment:
One instance of ws_store is included with ECL Watch, but you can choose to add another ws_store ESP Service...